core/userrights.class.inc.php

```diff
@@ -142,7 +142,9 @@ public function MakeSelectFilter($sClass, $aAllowedOrgs, $aSettings = array(), $
 		$oFilter  = new DBObjectSearch($sClass);
 		$oListExpr = ListExpression::FromScalars($aAllowedOrgs);
 
-		$oCondition = new BinaryExpression($oExpression, 'IN', $oListExpr);
+		$oNullCondition = new BinaryExpression($oExpression, 'IS', new ScalarExpression(null));
```

Comment:
Could you please use : 
		`$oNullCondition = new FunctionExpression('ISNULL', [$oExpression]);`
Which leads to the same result but is compliant with the OQL syntax.

Reply:
Better to make this as a suggestion: :wink:
```suggestion
		$oNullCondition = new FunctionExpression('ISNULL', [$oExpression]);
```